dfcreate.py

```diff
@@ -5,7 +5,7 @@
 spark = SparkSession.builder.master('local').getOrCreate()
 
 # Define data schema, then set data
-user_row = Row('birth','age','is_fan')
+user_row = Row('birth', 'age', 'is_fan')
```

Comment:
Please skip to review `dfcreate.py` in this PR. I want to refer this code in next PR, then will delete it